.gitignore

```diff
@@ -42,6 +42,7 @@ _UpgradeReport_Files/
 ipch/
 *.sdf
 *.opensdf
+*.VC.db
```

Comment:
you might want to PR this standalone if this a VC output file resulting from a node build